tests/test_extension.py

```diff
@@ -15,24 +15,30 @@ def test_get_default_config():
     assert "enabled = true" in config
 
 
-def test_get_config_schema():
+def test_config_schema_has_correct_keys():
+    """This is mostly a sanity check in case we forget to add a key."""
     ext = Extension()
 
     schema = ext.get_config_schema()
 
-    # Test the content of your config schema
-    assert "quality" in schema
-    assert "client_id" in schema
-    assert "client_secret" in schema
-    assert "lazy" in schema
+    assert set(schema.keys()) == {
```

Comment:
The original might be easier to understand on first glance. But I guess your changes ensure that a forgotten key will be caught.

Reply:
yes, it was more about ensuring that we *only* have the keys we think we have.  This test is largely pointless: it's mostly cheap insurance against forgetting to add a config in one place but requiring it in another.